src/Tes/Models/TesTask.cs

```diff
@@ -60,48 +60,48 @@ public TesTask()
         /// </summary>
         /// <value>Input files. Inputs will be downloaded and mounted into the executor container.</value>
         [DataMember(Name = "inputs")]
-        public List<TesInput> Inputs { get; set; }
+        public List<TesInput> Inputs { get; set; } = new List<TesInput>();
```

Comment:
`public List<TesInput> Inputs { get; set; } = new();` is very concise and does not eliminate any information, and is less distracting when reviewing. ditto for the others in this file.

Reply:
Or, possibly (see if the complier accepts it) `public List<TesInput> Inputs { get; set; } = [];`